dockers/skylab/HCA_post_processing/optimus_HCA_loom_merge.py

```diff
@@ -58,11 +61,10 @@ def main():
     input_id_list = []
     input_name_list = []
 
-    with loompy.new(args.output_loom_file) as dsout:
+    with loompy.new(output_loom_file) as dsout:
         for i in range(len(loom_file_list)):
             loom_file = loom_file_list[i]
             with loompy.connect(loom_file) as ds:
-                ds.ca['cell_names'] = ds.ca['cell_names'] + "-" + str(i)
                 # add input_id and input_name as column attributes
```

Comment:
This is the code needed to add the input_id as a column attribute. Needs to be commented in

Reply:
We changed the code in loom-output docker and now every file coming in to this step will have the input_id as a column attribute by default